lib/logger/index.ts

```diff
@@ -123,6 +125,20 @@ loggerLevels.forEach((loggerLevel) => {
   logger.once[loggerLevel] = logOnceFn as never;
 });
 
+if (is.string(process.env.LOG_FILE)) {
+  validateLogLevel(process.env.LOG_FILE_LEVEL);
+
+  // ensure log file directory exists
+  const directoryName = upath.parse(process.env.LOG_FILE).dir;
```

Comment:
```suggestion
  const directoryName = upath.dirname(process.env.LOG_FILE);
```
isn't that the same?